tools/WixBuild.props

```diff
@@ -105,7 +108,8 @@
     <PlatformSdkIncludePath Condition=" Exists('$(PlatformSdkIncludeRoot)\shared') ">$(PlatformSdkIncludePath);$(PlatformSdkIncludeRoot)\shared</PlatformSdkIncludePath>
     <PlatformSdkIncludePath Condition=" Exists('$(PlatformSdkIncludeRoot)\um') ">$(PlatformSdkIncludePath);$(PlatformSdkIncludeRoot)\um</PlatformSdkIncludePath>
     <PlatformSdkBinPath>$(PlatformSdkInstallPath)bin\</PlatformSdkBinPath>
-    <PlatformSdkBinPath Condition=" Exists('$(PlatformSdkBinPath)x86') ">$(PlatformSdkBinPath)x86\</PlatformSdkBinPath>
+    <PlatformSdkBinPath Condition=" Exists('$(PlatformSdkInstallPath)bin\x86') ">$(PlatformSdkInstallPath)bin\x86\</PlatformSdkBinPath>
+    <PlatformSdkBinPath Condition=" Exists('$(PlatformSdkInstallPath)bin\$(PlatformSdkBuildVersion)\x86') ">$(PlatformSdkInstallPath)bin\$(PlatformSdkBuildVersion)\x86\</PlatformSdkBinPath>
```

Comment:
D'oh. I'm getting out of this game. The repo reorg should take care of this. Worst case we can rely on VS command prompt or vswhere.